org.opent2t.sample.hub.superpopular/com.wink.hub/js/thingTranslator.js

```diff
@@ -102,16 +103,16 @@ class Translator {
     refreshAuthToken(authInfo) {
         var invalidAuthErrorMessage = "Invalid authInfo object.Please provide the existing authInfo object  with clientId + client_secret to allow the oAuth token to be refreshed";
         
-        if (authInfo == undefined || authInfo == null){
-            throw new Error(invalidAuthErrorMessage); 
+        if (authInfo === undefined || authInfo == null){
```

Comment:
Thanks! 

---
In reply to: [101367223](https://github.com/openT2T/translators/pull/158#discussion_r101367223) [](ancestors = 101367223,101101887)